entrypoint.sh

```diff
@@ -2,4 +2,4 @@
 
 set -eu
 export GIT_PR_RELEASE_TOKEN=$GITHUB_TOKEN
-git-pr-release
+git-pr-release $1
```

Comment:
https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action#writing-the-action-code